ext/RMagick/rmimage.c

```diff
@@ -12076,15 +12078,20 @@ resample(int bang, int argc, VALUE *argv, VALUE self)
 
     exception = AcquireExceptionInfo();
 #if defined(IMAGEMAGICK_7)
+    Image *preprocess = NULL;
```

Comment:
What is the difference between this and `Image *preprocess;` without the `NULL`?